frontend/src/bundles/common/components/video-modal/video-modal.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    ModalHeader,
+    ModalOverlay,
+} from '@chakra-ui/react';
+
+import { VideoModalContent } from './libs/components/components.js';
+
+type Properties = {
+    isOpen: boolean;
+    closeModal: () => void;
```

Comment:
```suggestion
    onModalClose: () => void;
```

https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md A9